iai_apartment/urdf/kitchen.xarco

```diff
@@ -796,28 +793,28 @@
 			</visual>
 			<collision>
 				<geometry>
-					<mesh filename="package://iai_apartment/meshes/handles/handle_horizontal.stl" />
+					<mesh filename="package://iai_apartment/meshes/handles/handle_horizontal.obj" />
 				</geometry>
 			</collision>
 		</link>
 
-<!-- Cabinets side B -->
-	<!-- Dishwasher: Cabinet 7 -->
```

Comment:
this comment, for example, will be very useful in future. I'd argue for the sake of maintainability, some of these comments must stay.